R/launchers.R

```diff
@@ -430,6 +455,47 @@ cluster_config <- function(
   list(command = "/bin/sh", args = args, rscript = rscript, quote = NULL)
 }
 
+#' Cloud Remote Launch Configuration
+#'
+#' Generates a remote configuration for launching daemons via cloud /
+#' cloud-based managed platforms.
+#'
+#' @param platform \[default "posit"\] character name of the platform
+#'   (case-insensitive). Currently the only option is "posit" to use the Posit
+#'   Workbench launcher.
+#'
+#' @inherit remote_config return
+#'
+#' @seealso [ssh_config()], [cluster_config()], and [remote_config()] for other
+#'   types of remote launch configuration.
+#'
+#' @examples
+#' tryCatch(cloud_config(), error = identity)
+#'
+#' \dontrun{
+#'
+#' # Launch 2 daemons using the Posit Workbench default:
+#' daemons(n = 2, url = host_url(), remote = cloud_config(platform = "posit"))
+#' }
+#'
+#' @export
+#'
+cloud_config <- function(platform = "posit") {
```

Comment:
Nit: `posit` is a bit too ambiguous, I'd suggest using `posit-workbench` as an identifier so it's clear that this doesn't work on e.g. Posit Connect.